.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Supports and troubleshooting
+    url: https://github.com/sequelize/sequelize/discussions
+    about: This issue tracker is not for bug reports which requires a SSCCE. Please refer to the bug report issue template.
+  - name: SSCCE
```

Comment:
Linking to the SSCCE from here isn't very useful. It's something they only need if they're reporting a bug and it's already part of the bug report template

Reply:
Thank you for the inputs!